### DISCUSSION TITLE
How to setup treblle adonisjs 6

### DISCUSSION THREAD

**Discussion by supakjack** [adonisjs/core#4492] [Help] [answered]
I follow guide in https://docs.treblle.com/en/integrations/adonisjs, and when ```node ace configure @treblle/adonisjs``` will get error ```[ error ] Package subpath './build/standalone' is not defined by "exports" in xxxx```

How to setup treblle adonisjs 6 ?

**Comment by RomainLanz** [comment, marked as answer]
Hey @supakjack! 👋🏻 

You are probably better off asking Treblle about that.
It does seems that they haven't updated their integration to work with AdonisJS 6 yet.

**Reply by supakjack** [reply]
Thank you, I'll go ask Treblle.
